library/src/main/java/com/rockerhieu/emojicon/EmojiconGridFragment.java

```diff
@@ -66,8 +67,7 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
             mData = People.DATA;
             mUseSystemDefault = false;
         } else {
-            Object[] o = (Object[]) getArguments().getSerializable("emojicons");
-            mData = Arrays.asList(o).toArray(new Emojicon[o.length]);
+            mData = (Emojicon[]) getArguments().getParcelableArray(EMOJICONS_KEY);
```

Comment:
@uwemaurer  Interesting. It worked fine when I tested it. Does it crash your app or something?